config/v2/eigendaflags/cli.go

```diff
@@ -128,6 +129,13 @@ func CLIFlags(envPrefix, category string) []cli.Flag {
 			Category: category,
 			Required: false,
 		},
+		&cli.StringFlag{
+			Name:     RegistryCoordinatorAddrFlagName,
+			Usage:    "Address of the registry coordinator contract.",
+			EnvVars:  []string{withEnvPrefix(envPrefix, "REGISTRY_COORDINATOR_ADDR")},
+			Category: category,
+			Required: false,
+		},
```

Comment:
this can be fetched from the ServiceManager right?

Reply:
IIRC the `Reader` struct doesn't expose this, and would need to be modified to do so.

Reply:
[df6273a](https://github.com/Layr-Labs/eigenda-proxy/pull/384/commits/df6273a049b984a44254fe2c7c412d3b3b19219c)